Enhanced_Zeus_Modules.sqf

```diff
@@ -11036,6 +11036,147 @@ MAZ_EZM_fnc_initFunction = {
+				"a3\ui_f\data\igui\cfg\holdactions\holdaction_search_ca.paa", 
+				"_this distance _target < 3",
+				"_caller distance _target < 3",  
+				{},
+				{},
+				{
+					private _intel = _this select 0; 
+					private _args = _this select 3;
+					_args params ["_title","_description","_deleteOnPickup", "_visibility"];
+
+					[_intel,0] remoteExec ["removeAction",0,true];
+					["IntelAdded",[format ["%1 picked up %2", (name _caller), _title], "a3\ui_f\data\igui\cfg\simpletasks\types\documents_ca.paa"]] remoteExec ['BIS_fnc_showNotification',0];
+
+					comment "for all players, remoteExec the createDiaryRecord";
+
+					private _intelParams = ["Diary", [format["%1", _title], format["%1", _description]]];
+
+					{
+						[_x, _intelParams] remoteExec ["createDiaryRecord", _x];
+					} forEach allPlayers;
+
+					if (_deleteOnPickup) then {
+						deleteVehicle _intel;
+					};
+				},
+				{},
+				[_title, _description, _deleteOnPickup, _visibility],
+				_holdDuration, 
+				10, 
+				true,
+				false 
+			];
+			_actionParams remoteExec ["BIS_fnc_holdActionAdd", 0, _intelObj];
+
+			[_intelObj] call MAZ_EZM_fnc_addObjectToInterface;
+			["Intel created.","addItemOk"] call MAZ_EZM_fnc_systemMessage;
+		};
+
+		HYPER_EZM_fnc_createIntel = {
+				private _dialogTitle = "Create Intel";
+				private _target = [true] call MAZ_EZM_fnc_getScreenPosition;
+				private _content = [
+					[
+						"EDIT",
+						"Title",
+						[
+								"",
+								1
+						]
+					],
+					[
+						"EDIT",
+						"Description",
+						[
+								"",
+								3
+						]
+					],
+					[
+						"COMBO",
+						"Intel Object",
+						[
+								["documents", "laptop", "ruggedtablet", "tablet", "folder"],
+								["Documents", "Laptop", "Rugged Tablet", "Tablet", "Folder"],
+								0
+						]
+					],
+					[
+						"TOOLBOX",
+						"Delete on Pick Up",
+						[
+								true,
+								["No", "Yes"]
+						]
+					],
+					[
+						"SLIDER",
+						"Hold Duration",
+						[
+							1,
+							12,
+							3,
+							objNull,
+							[1,1,1,0.7],
+							false
+						]
+					],
+					[
+						"COMBO",
+						"Show Intel To",
+						[
+								["group", "side", "global"],
+								["Group", "Side", "Global"],
+								1
+						]
+					]
+
+				];
+				private _onConfirm = {
+						params ["_values", "_args", "_display"];
+						_values params ["_title","_description","_intelObject","_deleteOnPickup","_holdDuration", "_visibility"];
+						private _target = _args # 0;
+						[ _title, _description, _intelObject, _deleteOnPickup, _target, _holdDuration, _visibility ] call HYPER_EZM_fnc_handleCreateIntel;
+						_display closeDisplay 1;
+				};
+				private _onCancel = {
+						params ["_values", "_args", "_display"];
+						systemChat "Create Intel dialog canceled.";
+						_display closeDisplay 2;
+				};
+				[
+						_dialogTitle,
+						_content,
+						_onConfirm,
+						_onCancel,
+						[_target]
```

Comment:
This doesn't need to be an array. You can pass the `_target` variable directly into `_args`.